pkg/registry/replacesgraphloader.go

```diff
@@ -15,11 +15,21 @@ func (r *ReplacesGraphLoader) CanAdd(bundle *Bundle, graph *Package) (bool, erro
 		return false, fmt.Errorf("Invalid content, unable to parse bundle")
 	}
 
+	// adding the first bundle in the graph
+	if replaces == "" {
```

Comment:
this is duplicated

Reply:
nice catch - must have happened when I reverted this file...